.gitignore

```diff
@@ -8,10 +8,7 @@
 /build
 
 # scripts
-/scripts/conduits/data/
-raidbotsConduitsInfo.json
-soulbindconduititem.json
-soulbindconduitrank.json
+/scripts/talents/*.json
```

Comment:
Should the generated TS files be included in the gitignore as well?

Reply:
No - the idea is that we know what information is available for everywhere, but anyone can run the script easily to update between patches